test/functionalTest/cases/0979_patch_v2_entities_eid/patch_v2_entities_eid.test

```diff
@@ -35,7 +35,8 @@ brokerStart CB
 # 03. GET /v2/entities/E1
 # 04. PATCH /v2/entities/E1 attr1=4
 # 05. GET /v2/entities/E1 - see attr1=4 AND attr2=2 (untouched)
-# 06. PATCH /v2/entities/E1 attr3=6 - error, as attr3 doesn't exist
+# 06. PATCH /v2/entities/E1 attr3=6
+# 07. GET /v2/entities/E1 - see attr1, attr2 and attr3
```

Comment:
attr3 will not be shown, after rollback (probably the step 07 could be removed, to revert this .test to its original form).